modules/websocket/websocket_server.h

```diff
@@ -41,6 +41,8 @@ class WebSocketServer : public WebSocketMultiplayerPeer {
 	GDCLASS(WebSocketServer, WebSocketMultiplayerPeer);
 	GDCICLASS(WebSocketServer);
 
+	String bind_ip = "*";
```

Comment:
Please use `IP_Address` type instead of string.
Also initialize it's value in the constructor, not in the header (for consistency with the rest of the module until we officially move to newer version of C++).

Reply:
Unless i miss sth important IP_Adress class can not be used from GDScript and thus i refrained from using it in order to make host settings changeable from script.
(WebSocketServer itself is usable from GDScript)
Please tell me if i was mistaken in my analysis of the situation.

Reply:
should be fixed now